sdf/how-to/_category_.json

```diff
@@ -1,5 +1,5 @@
 {
   "label": "How To",
-  "collapsed": false,
+  "collapsed": true,
```

Comment:
I think this should be collapsed by default otherwise it pushes everything else to the bottom of the page